pages/common/iperf3.md

```diff
@@ -0,0 +1,23 @@
+# iperf3
+
+> Iperf3 — traffic generator for testing network bandwidth
+
+- Run iperf3 as a server
+
+`iperf3 -s`
+
+- Run iperf3 server on port 5209
+
+`iperf3 -s -p5209`
+
+- Start bandwidth test
+
+`iperf3 -c {{server}}`
+
+- Start bandwidth test in 5 threads
+
+`iperf3 -c {{server}} -P5`
+
+- Start bandwidth test in reverse mode
```

Comment:
What is reverse mode? 🤔

Reply:
Reverse mode means a different direction of the test. The server sends data to the client.
Perhaps in this case it's better to write `Reverse direction of the test. Server sends data to the client:`?